rules/solid/src/Rector/If_/ChangeAndIfToEarlyReturnRector.php

```diff
@@ -122,9 +122,9 @@ public function refactor(Node $node): ?Node
         $this->addNodesAfterNode($ifs, $node);
         $this->addNodeAfterNode($ifReturn, $node);
 
-        $ifParentReturn = $this->getIfParentReturn($node);
-        if ($ifParentReturn !== null) {
-            $this->removeNode($ifParentReturn);
+        $ifNextReturn = $this->getIfNextReturn($node);
+        if ($ifNextReturn !== null && !$this->isIfInLoop($node)) {
```

Comment:
stripping the `return` is still required for https://github.com/clxmstaab/rector/blob/fix-expect/rules/solid/tests/Rector/If_/ChangeAndIfToEarlyReturnRector/Fixture/only_one_if_before_class_method_return.php.inc